tasks.py

```diff
@@ -25,6 +25,7 @@
         "--ignore=network-api/networkapi/wagtail_l10n_customization/*",
         "--ignore=network-api/networkapi/settings.py",
         "--ignore=network-api/networkapi/wagtailpages/__init__.py",
+        "--ignore=network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_reporter_page.html",
```

Comment:
This line is from Theo. See: https://github.com/mozilla/foundation.mozilla.org/issues/5081#issuecomment-685759249